benchexec/tablegenerator/__init__.py

```diff
@@ -697,6 +695,11 @@ def read_logfile_lines(log_file):
 
                     value = get_value_from_logfile(logfileLines, column.pattern)
 
+            if column.title.lower() == 'score':
+                if value is None:
+                    # If no score column exists in the xml, take the internally computed score,
+                    # if available
+                    value = str(score) if score is not None else None
```

Comment:
Can be simplified to:
```suggestion
            if column.title.lower() == 'score' and value is None and score is not None:
                # If no score column exists in the xml, take the internally computed score, if available
                value = str(score)
```

Please also check the failing test.

Reply:
That's way nicer, thanks for the hint!

Reply:
The `if` in the assignment can be removed.

Reply:
Please fix the indentation, too.

Reply:
D'oh, sorry.